esi_leap/objects/contract.py

```diff
@@ -62,6 +68,20 @@ def get_all_by_status(cls, context, status):
             context, status)
         return cls._from_db_object_list(context, db_contracts)
 
+    # @classmethod
```

Comment:
Can we just remove all of these commented-out lines?

Reply:
Yep!